### PR TITLE
Adding my Canvas Creation to repo

### PR DESCRIPTION
**Pull request by krdormer**
Submission for Canvas Creations - Hacktoberfest 2020 @aneagoie

### CODE REVIEW

**Comment by MattCSmith** [comment]
Hey @krdormer, I am unable to resolve the merge conflict on my end. 
Kindly resolve the conflict so we are able to merge it in.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,79 @@
-.DS_Store
```
Comment (kennithnichol) [inline .gitignore]: I don't think we should be replacing the project's .gitignore file here.
You should consider setting your global gitignore to keep your OS and IDE files out of the repository.

Typically the project gitignore would include .DS_Store and Thumbs.db and then any project related exclusions (for example the node_modules or build folders).

reference:
https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#create-a-global-gitignore

**Review by kennithnichol — CHANGES_REQUESTED** [review]
I'm not sure why there are so many other users contributions in your commit here.
Perhaps you could try update the PR with the latest from our master?

**Comment by krdormer** [comment]
Apologies, I attempted to submit my work as outlined in the instructions, could I get some clarity on what the conflict is here? 

I see an Art/.DS_Store, and the include.js, but I am not sure how to resolve those conflicts. 

Thank you for any help you may have!

**Comment by sophiabrandt** [comment]
@krdormer what exactly is your creation? It looks like you change files in several folders?
If possible, please only add your `Art` folder and adjust `include.js`. `include.js` has syntax errors, you'll need to fix them.
Also, remove `.DS_Store` files.

**Comment by MattCSmith** [comment]
Closing due to age and/or conflicts, feel free to reopen if required. I have also marked this as accepted for the Hacktoberfest challenge as the effort was made.

**Pull request closed without merging**